devtron-labs/dashboard
# fix: Prevent saving with empty project name

# Description

Global config > Project > Prevent saving with empty project name

![image](https://user-images.githubusercontent.com/68757539/219646918-eefb6b26-1519-49d6-8354-705fab27ac24.png)


Fixes [AB 2060](https://dev.azure.com/DevtronLabs/Devtron/_boards/board/t/OSS%20adoption/Stories/?workitem=2060)

## Type of change


# How Has This Been Tested?

1. Go to Global Config > Projects
2. Click on add project
3. Click on Save without providing a project name
4. Clicking on save without providing a Project name should show a field validation error with message "This is a required field"



# Checklist:
